maze-game/Player.java

```diff
@@ -412,6 +451,30 @@ private void resetSpeed()
         useSpeedPotion = false;
         activateSpeedPotion = false;
     }
+    
+    private void toggleMagnet()
+    {
+        if(magState == false)
+        {
+            if(MyWorld.score != 0)
+            {
+                MyWorld.score -= 1;
```

Comment:
why is the score reduced here

Reply:
so that they can't keep spamming m and using the magnet for free

Reply:
oh